### PR TITLE
Fix kubernetes client bump

### PR DESCRIPTION
**Pull request by rosalieper**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # api
 
+## 8x.9.8 - 01 September 2022
```
Comment (deer-wmde) [inline CHANGELOG.md:3]: errr... somehow this file ended up in here after rebasing oO

**Comment by rosalieper** [comment]
LGTM the previously failing test passed for me locally. I had this error, tho I think is unrelated `1) Tests\Jobs\PlatformStatsSummaryJobTest::testQueryGetsStats
PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'dbname1.dbprefix1_site_stats' doesn't exist
`

**Comment by deer-wmde** [comment]
Yeah some of our tests on `main` fail for me locally too, it's probably not related to this PR. What is interesting is that the tests pass on the github CI runner though: https://github.com/wbstack/api/actions/runs/3490179331/jobs/5841270884#step:8:12

I wonder why, because I think the same docker image is used :thinking: I created https://github.com/wbstack/api/issues/549

File: CHANGELOG.md (hunk 1 above)
Comment (tarrow) [reply CHANGELOG.md:3]: yeah, I'm interested how you have this commit locally but it's missing in main. I guess we'll just be naughty and merge it here anyway

**Review by tarrow — APPROVED** [review]

**Pull request merged**